modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -118,5 +127,30 @@ def enum_zones
       logger.debug "Enumerated dns zones: #{zones}"
       zones
     end
+
+    def get_ptr_ip(zone, ip)
+      ip6 = ip.include?(':')
+      if ip6
+        zone.slice!('.ip6.arpa')
+        ip6_zone = []
+        partial_ip6 = ''
+        zone.split('.').reverse_each do |part|
+          partial_ip6 += part
+          if partial_ip6.length == 4
+            ip6_zone << partial_ip6
+            partial_ip6 = ''
+          end
+        end
+        ip6_zone + [partial_ip6] unless partial_ip6.empty?
+      end
+      if ip6
+        ip.slice!("#{ip6_zone.join(':')}:")
+        result = "#{ip.split(':').reverse.join(':')}"
```

Comment:
Right! I could not find any documentation. Turns out we can use the full reverse output of the IP to add the record. However, I cannot point to any documentation on this.